src/sortStudents.ts

```diff
@@ -1,16 +1,57 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const copyStudents = [...students];
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+  function calculateAverage(grades: number[]):number {
+    return grades.reduce((sum, value) => sum + value, 0) / grades.length;
+  }
+
+  copyStudents.sort((a: Student, b: Student) => {
+    let direction = -1;
+
+    if (order === 'asc') {
+      direction = 1;
+    }
```

Comment:
```suggestion
    const direction = order === 'asc' ? 1 : -1;
```

Better use ternary here.